dpdata/plugins/ase.py

```diff
@@ -140,7 +140,7 @@ def from_multi_systems(
         frames = ase.io.read(file_name, format=ase_fmt, index=slice(begin, end, step))
         yield from frames
```

Comment:
I guess you should use ase.io.iread or else yield from is meaningless. I will just load all data into memory.